### DISCUSSION TITLE
Does SeisSol support different base functions for different regions?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#1060] [Q&A] [answered]
For example, in the [WP2_LOH1](https://github.com/SeisSol/Examples/tree/master/WP2_LOH1) example, can I set up a 4th order basis function for Volume1 and a 6th order basis function for Volume2？

**Comment by Thomas-Ulrich** [comment, marked as answer]
no

**Reply by Thomas-Ulrich** [reply]
(this is an intended future feature).
